README.md

```diff
@@ -169,6 +169,18 @@ Optionally, if you wish to lift your server in test mode, you can pass the envir
 sails lift --environment=test
 ```
 
+####### Changing name of environment
+
+If you run your tests with another environment variable name, you can specity it in the configuration:
+
+```javascript
+module.exports.rewire = {
+  environment: 'shippable'
```

Comment:
This would mean that only one single environment other than `test` can be used to run rewire. How about something more generic -

```
module.exports.rewire = {
  environments: {
    test: true,  // should default to true
    shippable: true,
    development: false 
  }
```